lib/senkyoshi.rb

```diff
@@ -103,7 +111,7 @@ def self.initialize_course(canvas_file_path, blackboard_file_path)
     end
   end
 
-  def true?(obj)
+  def self.true?(obj)
```

Comment:
```ruby
module Foo
  def howdy?
    "hi"
  end
end

module Foo
  class Bar
    include Foo # for instance methods
    extend Foo # for class methods

    def meclass
      howdy?
    end

    def self.meinstance
      howdy?
    end
  end
end
```

Just wasn't a need to have it a class method. I didn't realize you were using it in class methods. So if you feel this is cleaner then that is fine with me.